Automattic/wp-calypso
# Hosting Command Palette: Add banner on /sites

- Related to https://github.com/Automattic/dotcom-forge/issues/4409

## Proposed Changes

* Affordance banner to promote the Hosting Command Palette on `/sites`

## Testing Instructions



Mobile

* Navigate to `/sites` on a mobile phone
* Observe there is no new banner

Reduce motion

* Navigate to `/sites` on Desktop
* Set your Mac `Preferences > Accessibility > Display > Reduce motion` toggle enabled.
* Observe there is no animation
* Set your Mac `Preferences > Accessibility > Display > Reduce motion` toggle disabled.
* Observe there is animation

Happy path desktop

* Navigate to `/sites` on your desktop
* Observe the new banner. On Mac it displays `⌘K` on other platforms `Ctrl+K`
* Click on the X button to close the banner
* Recharge the page and observe the banner doesn't appear anymore

## Screenshots

Mac:
<img width="1184" alt="Screenshot 2023-12-20 at 19 22 36" src="https://github.com/Automattic/wp-calypso/assets/779993/64e8a8bb-471a-4d05-97ac-ea4b96dbadbb">

Other platforms:
<img width="1176" alt="Screenshot 2023-12-20 at 19 23 24" src="https://github.com/Automattic/wp-calypso/assets/779993/8538b62d-2811-4a04-901d-51bef2156f1f">

"Hidden" on touchable screens:

<img width="1185" alt="Screenshot 2023-12-20 at 20 54 44" src="https://github.com/Automattic/wp-calypso/assets/779993/eb8d90cf-de32-4d18-8e59-2c278bfa9ca9">

## Screencast

https://github.com/Automattic/wp-calypso/assets/779993/8d1c5d5a-b7da-44f8-94dc-4557ac9bd4d9


https://github.com/Automattic/wp-calypso/assets/779993/55b6529c-8ba7-41ad-b900-160921b8ff55






## Pre-merge Checklist
